### PR TITLE
added username, password, sign in and some styling to main page - no …

### PR DESCRIPTION
**Pull request by SaaraLeppis**
…functionality

### CODE REVIEW

**Comment by SaaraLeppis** [comment]
Added the labels and inputs and button to front page. + styling. No functionalities. 
Please check and merge if ok.

File: src/components/Main.js [hunk 1]
```diff
@@ -2,10 +2,24 @@ import React from "react";
...
-    <div>
-      <h1>Dance</h1>
-      <h1>Moves</h1>
-      This is the Main component
+    <div className="main-area">
+      <div className="main-text">
+        <h1 className="h1-dance">Dance</h1>
+        <h1 className="h1-move">Move</h1>
+      </div>
+      <div className="login">
```
Comment (Termanty) [inline src/components/Main.js]: I think this login div should be form-tag.

**Comment by NooraJumppanen** [comment]
Looks ok!

File: src/components/Main.js [hunk 2]
```diff
@@ -2,10 +2,24 @@ import React from "react";
 
 const Main = () => {
   return (
-    <div>
-      <h1>Dance</h1>
-      <h1>Moves</h1>
-      This is the Main component
+    <div className="main-area">
```
Comment (Termanty) [inline src/components/Main.js:5]: this is comment

**Pull request merged**